acquia/acquia_cms
# ACMS-708: colors to pallete and also changed the colors on the links.

**Motivation**
Accessibility issues identified by running pa11y test on /home URL.
Fixes #708

**Proposed changes**
- Added New Colors to the Palette
- Using New colors changed 2 Custom styles that fixes all the accessibility issues related to homepage  (Check screenshot below).

**Alternatives considered**
-  Following changes affects only the homepage of ACMS, all other pages contain colors applied using old colors in the pallete, they will have to be updated to new color added.

**Testing steps**
- Install pa11y on your local machine using node - `npm install -g pa11y`
- Once installed, shift to branch ACMS-708 and run the following example command - `pa11y http://localhost:8888/acquia_cms/docroot/home`
- Notice that there are no issues observed after shifting to this branch(see attached screenshot below).

**Merge requirements**
- [* ] Manual testing by a reviewer

<img width="1069" alt="Screenshot 2021-04-07 at 10 54 28 PM" src="https://user-images.githubusercontent.com/15887127/113908648-66f3cc00-97f4-11eb-8884-2c158b5cf0d2.png">
